tools/health/isAlive

```diff
@@ -85,13 +85,6 @@ def isElkAlive() :
     okString = '"cluster_name"'
     return monitorUtil.outputChecker(cmd, okString, 1)
 
-def isKafkaAlive() :
```

Comment:
Why do we need a way to check that? I do believe our testsuite is making sure enough that Kafka is fine?